src/python/pants/backend/python/subsystems/repos.py

```diff
@@ -60,6 +63,42 @@ class PythonRepos(Subsystem):
         advanced=True,
     )
 
+    path_mappings = StrListOption(
+        help=softwrap(
+            f"""
+            Mappings to facilitate using local Python requirements when the absolute file paths
+            are different on different users' machines. For example, the
+            path `file:///Users/pantsbuild/prebuilt_wheels/django-3.1.1-py3-none-any.whl` could
+            become `file://${{WHEELS_DIR}}/django-3.1.1-py3-none-any.whl`, where each user can
+            configure what WHEELS_DIR points to on their machine.
+
+            Expects values in the form `NAME|PATH`, e.g.
+            `WHEELS_DIR|/Users/pantsbuild/prebuilt_wheels`. You can specify multiple
+            entries in the list.
+
+            This feature is intended to be used with `[python-repos].find_links`, rather than PEP
+            440 direct reference requirements (see
+            {doc_url("python-third-party-dependencies#local-requirements")}.
+            `[python-repos].find_links` must be configured to a valid absolute path for the
+            current machine.
+
+            Tip: you can avoid each user needing to manually configure this option and
+            `[python-repos].find_links` by using a common file location, along with Pants's
+            interpolation support ({doc_url('options#config-file-interpolation')}. For example,
+            in `pants.toml`, you could set both options to `%(buildroot)s/python_wheels`
+            to point to the directory `python_wheels` in the root of
+            your repository; or, use the path `%(env.HOME)s/pants_wheels` for the path
+            `~/pants_wheels`. If you are not able to use a common path like this, then we
+            recommend setting that each user set these options via a `.pants.rc` file
+            ({doc_url('options#pantsrc-file')}.
+
+            Note: Only takes effect if you use Pex lockfiles. Use the default
```

Comment:
Why mention it if it's the default and soon to be only lockfile type? More info is not always better, and I think this is adding cognitive load.  In fact, "Note: only takes effect when using lockfiles." with a link to the docs page, is enough.

Reply:
I copied this from your `only_binary` help message ;)

Reply:
Which I probably copied from some other place. The rot stops here! :)